spec/index.adoc

```diff
@@ -1637,11 +1637,13 @@ For security reasons, only HTTPS connections are allowed (URLs must start with `
 
 The maximal size of a request URL is 2048 bytes.
 
-The maximal size of a response is `2MiB`. Therefore, only the first `2MiB` will be returned if a response is larger than this size. This size limit also applies to the value returned by the `transform` function.
+Each request should specify the maximal expected size for the response. The upper limit on the size of a response is `2MiB`. Therefore, only the first number of bytes as specified in the request (up to 2MiB), will be returned if a response is larger than these limits. The 2MiB size limit also applies to the value returned by the `transform` function.
```

Comment:
Shall we specify the unit expected for this field? It feels like `KB` is the intended unit, in that case, we probably want to explain the upper limit in targeted unit.

Reply:
... the way I read it, the unit is `bytes`.

Reply:
But, yes, we should make it more explicit.

Reply:
please see updated text